superset-frontend/src/components/FilterableTable/index.tsx

```diff
@@ -114,13 +92,10 @@ const StyledFilterableTable = styled.div`
       overflow: hidden;
     }
 
-    .ReactVirtualized__Table__headerColumn,
-    .ReactVirtualized__Table__rowColumn,
-    .grid-cell {
+    .ant-table-cell,
+    .virtual-table-cell {
```

Comment:
@justinpark @EugeneTorap Can you provide some context why the default Table theming is not sufficient? The reason I'm asking is because one of the main reason we have a table component is to make sure all tables look and behave the same throughout the application. What are the behaviors that the default Table is missing? Can we implement them on the default Table?

Reply:
Good point. There's a bunch of differences between this update and existing one.

| |screenshot|
|--|--|
|Existing one|<img width="945" alt="Screenshot 2023-05-05 at 3 27 30 PM" src="https://user-images.githubusercontent.com/1392866/236579091-dfec438d-4313-41b0-82d7-2be037760505.png">
|Table component (no stripes, bigger font size, bigger padding)|<img width="825" alt="Screenshot 2023-05-05 at 3 28 52 PM" src="https://user-images.githubusercontent.com/1392866/236579112-b9c40c45-b2ee-4062-a54b-b9acc492ed73.png">|
|Table with custom style (no stripes, bigger padding but same font size)|<img width="942" alt="Screenshot 2023-05-05 at 3 28 14 PM" src="https://user-images.githubusercontent.com/1392866/236579496-af24ce89-898e-4d5c-a7e3-b2275cc9bfd2.png">|

The major style update is font-size since it impacts the number of visible rows due to increased row height.
I can add this custom style on default Table if applicable for other cases.

Reply:
I like how `Table with custom style` looks!

Reply:
For me, the important part is that tables look consistent. So I'm ok with either option as long as it's applied globally to the table component and not only to SQL Lab. To make this type of decision, we need to consider the table use cases globally. I think @kasiazjc should be the one to decide which style is more appropriate.